doc/src/examples.rst

```diff
@@ -18,40 +19,34 @@ Euler Equations
 Proceed with the following steps to run a parallel 2D Euler vortex
 simulation on a structured mesh:
 
-1. Create a working directory called ``euler_vortex_2d/``
+1. Navigate to the ``PyFR-Test-Cases/2d-euler-vortex`` directory::
 
-2. Copy the configuration file
-   ``PyFR/examples/euler_vortex_2d/euler_vortex_2d.ini`` into
-   ``euler_vortex_2d/``
+        cd PyFR-Test-Cases/2d-euler-vortex
 
-3. Copy the `Gmsh <http:http://geuz.org/gmsh/>`_ file
-   ``PyFR/examples/euler_vortex_2d/euler_vortex_2d.msh`` into
-   ``euler_vortex_2d/``
+2. Run pyfr to convert the `Gmsh <http:http://geuz.org/gmsh/>`_
+   mesh file into a PyFR mesh file called ``2d-euler-vortex.pyfrm``::
 
-4. Run pyfr to convert the `Gmsh <http:http://geuz.org/gmsh/>`_
-   mesh file into a PyFR mesh file called ``euler_vortex_2d.pyfrm``::
+        pyfr import 2d-euler-vortex.msh 2d-euler-vortex.pyfrm
 
-        pyfr import euler_vortex_2d.msh euler_vortex_2d.pyfrm
+3. Run pyfr to partition the PyFR mesh file into two pieces::
 
-5. Run pyfr to partition the PyFR mesh file into two pieces::
+        pyfr partition 2 2d-euler-vortex.pyfrm .
```

Comment:
I think some of these names are still wrong compared with what we have in the test case repo.

Reply:
I built the docs and copy pasted all commands into the latest version of the tests repo and it all ran fine. So I think it is all good. I'll merge.